packages/workflow/src/WorkflowDataProxy.ts

```diff
@@ -922,17 +922,14 @@ export class WorkflowDataProxy {
 					throw createExpressionError(`"${nodeName}" node doesn't exist`);
 				}
 
-				if (!that?.runExecutionData?.resultData?.runData.hasOwnProperty(nodeName)) {
-					throw createExpressionError(`no data, execute "${nodeName}" node first`);
-				}
```

Comment:
is this change needed in the same PR? If yes, I think we should add a little note in the PR description why we stopped throwing expression errors in this case.
That would be really helpful for future us.

Reply:
Sadly yes. Without that change, the code would not work as it would error instead of returning `false` and we would bo again where we started and why we make that change.